packages/nextra-theme-docs/src/styles.css

```diff
@@ -558,58 +516,6 @@ input[type='search']::-webkit-search-results-decoration {
   @apply mr-1;
 }
 
-/* RTL */
-.nextra-container.rtl {
-  direction: rtl;
-  .nextra-sidebar ul ul {
-    @apply mr-5 ml-0;
-  }
-  .nextra-sidebar button,
-  .nextra-sidebar a {
-    @apply text-right;
-  }
-  article blockquote {
-    @apply border-r-2 border-l-0 pr-6 pl-0;
-  }
-}
-
-@screen md {
-  .nextra-container:not(.page) article .bleed.full {
-    margin-left: calc(-50vw + 50% + 8rem);
-    margin-right: calc(-50vw + 50% + 8rem);
-  }
-  .nextra-container.page article .bleed.full {
-    margin-left: calc(-50vw + 50%);
-    margin-right: calc(-50vw + 50%);
-  }
-}
-@screen xl {
-  .nextra-container:not(.rtl):not(.page) article .bleed.full {
-    margin-left: calc(50% - 50vw + 16rem);
-    margin-right: calc(50% - 50vw);
-  }
-  .nextra-container.rtl:not(.page) article .bleed.full {
-    margin-left: calc(50% - 50vw);
-    margin-right: calc(50% - 50vw + 16rem);
-  }
-  .nextra-container.page article .bleed.full {
-    margin-left: calc(50% - 50vw);
-    margin-right: calc(50% - 50vw);
-  }
-}
```

Comment:
moved bleed's styles directly in `bleed.tsx`, except styles with `.page` class as he no longer exist
https://github.com/shuding/nextra/blob/822c98aae2c4e5ce71b70fd4becc8c1a7a43fbf8/packages/nextra-theme-docs/src/components/bleed.tsx#L15-L19